.babelrc

```diff
@@ -10,13 +13,14 @@
       ]
     },
     "development": {
-      "plugins": [
-        "rewire"
-      ]
-  },
+      "plugins": []
+    },
     "test": {
       "plugins": [
-        "rewire"
+        "rewire",
+        ["babel-plugin-transform-require-ignore", {
```

Comment:
Why is this required for tests but not for other builds?

Reply:
Parcel handles the scss/image imports but babel doesn't know how to do that (for the karma test suites). We also don't need those to be loaded for the tests so we ignore them.